src/status_im/events.cljs

```diff
@@ -115,30 +115,39 @@
 (fx/defn system-theme-mode-changed
   {:events [:system-theme-mode-changed]}
```

Comment:
could you please elaborate how reopening the app related to system-theme-mode-changed ?

Reply:
Hi @flexsurfer,
when the app reopened Appearance module fired the theme change event **twice** (even though the theme was the same), due to a [bug](https://github.com/facebook/react-native/issues/28525) in the react-native library.
So when the system theme change event fired, it called the `system-theme-mode-changed` event two times.
And in issue, we also noticed there was a back button on the home screen view when this happened. It happened because we were calling the `navigate-to` event also for view on the home screen.
We never observed this behavior before this module bug, Because in android calling `navigate-to` for view on the home screen had no effect, and in ios also it is showing the back button only after the bug is firing the theme change event twice.
So, in this change, we also make sure even if some reason event fires twice, we should not navigate to the screens, which are already on the home screen.

Reply:
but i'm wondering why we should handle this event at all if the theme is not changed?